server/tests/src/test/java/org/infinispan/server/functional/overview/OverviewReportIT.java

```diff
@@ -0,0 +1,68 @@
+package org.infinispan.server.functional.overview;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.infinispan.util.concurrent.CompletionStages.join;
+
+import java.util.Map;
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.rest.RestClient;
+import org.infinispan.client.rest.RestResponse;
+import org.infinispan.commons.dataconversion.internal.Json;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.rest.assertion.ResponseAssertion;
+import org.infinispan.server.test.junit5.InfinispanServerExtension;
+import org.infinispan.server.test.junit5.InfinispanServerExtensionBuilder;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import net.spy.memcached.ConnectionFactoryBuilder;
+import net.spy.memcached.MemcachedClient;
+
+public class OverviewReportIT {
```

Comment:
I would not introduce a new IT for this, but add this to the `RESTAuthorizationTest` class so that it gets properly tested with authorization.

Reply:
The test uses REST, but also HotRod and the Memcached protocol to see the effects on the clients report.
Moving it to `RESTAuthorizationTest` I wasn't able to send a put using the Memcached client, so I had to remove the check for the `Memcached` client on the report.
I did the change in a different commit.

Reply:
It seems that it is not possible to safely merge the test into the `RESTAuthorizationTest`.
I have another idea , keep the test since we need to use HotRod and Memcached protocol from it and add a simpler form of it into `RESTAuthorizationTest` that avoids to use other protocols.

Reply:
Specifically, `OverviewReportIT` will keep to test the fact that usages of clients are reflected in the report.
While the test I'm adding to `RESTAuthorizationTest` will test the authorization behavior of the REST call. 
I don't know what you think of it.